### ISSUE TITLE
computing dtt error (indexing) with latest pandas

### ISSUE THREAD

**Issue by ThomasLecocq** [ROBelgium/MSNoise#210]
TravisCI fails with

```
======================================================================
ERROR: test_025_dtt (__main__.MSNoiseTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/miniconda/lib/python3.7/site-packages/pandas/core/series.py", line 1002, in __setitem__
    self._set_with_engine(key, value)
  File "/home/travis/miniconda/lib/python3.7/site-packages/pandas/core/series.py", line 1033, in _set_with_engine
    loc = self.index._engine.get_loc(key)
  File "pandas/_libs/index.pyx", line 70, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/index.pyx", line 75, in pandas._libs.index.IndexEngine.get_loc
TypeError: '[ 0  1  2  3  4  5  6  7  8  9 10 11 12 13 14 15 23 31 32 33 34 35 36 37 38
 39 40 41 42 43 44 45 46]' is an invalid key
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/home/travis/miniconda/lib/python3.7/site-packages/msnoise-1.6-py3.7.egg/msnoise/test/tests.py", line 312, in test_025_dtt
    main()
  File "/home/travis/miniconda/lib/python3.7/site-packages/msnoise-1.6-py3.7.egg/msnoise/s06compute_dtt.py", line 231, in main
    df['err'][tmp] = 1.0
  File "/home/travis/miniconda/lib/python3.7/site-packages/pandas/core/series.py", line 1026, in __setitem__
    self._set_with(key, value)
  File "/home/travis/miniconda/lib/python3.7/site-packages/pandas/core/series.py", line 1061, in _set_with
    self.loc[key] = value
  File "/home/travis/miniconda/lib/python3.7/site-packages/pandas/core/indexing.py", line 666, in __setitem__
    indexer = self._get_setitem_indexer(key)
  File "/home/travis/miniconda/lib/python3.7/site-packages/pandas/core/indexing.py", line 617, in _get_setitem_indexer
    return self._convert_to_indexer(key, axis=0, is_setter=True)
  File "/home/travis/miniconda/lib/python3.7/site-packages/pandas/core/indexing.py", line 1198, in _convert_to_indexer
    return self._get_listlike_indexer(key, axis, raise_missing=True)[1]
  File "/home/travis/miniconda/lib/python3.7/site-packages/pandas/core/indexing.py", line 1254, in _get_listlike_indexer
    self._validate_read_indexer(keyarr, indexer, axis, raise_missing=raise_missing)
  File "/home/travis/miniconda/lib/python3.7/site-packages/pandas/core/indexing.py", line 1304, in _validate_read_indexer
    raise KeyError(f"{not_found} not in index")
KeyError: '[1, 2, 3, 4, 6, 7, 8, 9, 11, 12, 13, 14, 23, 31, 32, 33, 34, 36, 37, 38, 39, 41, 42, 43, 44, 46] not in index'
----------------------------------------------------------------------
Ran 51 tests in 97.499s
```

which is linked to ``df['err'][tmp] = 1.0`` ; I guess should be using ``.iloc[...]`` instead. 

Indentical to to #209

**Comment by ThomasLecocq** [comment]
https://travis-ci.org/github/ROBelgium/MSNoise/jobs/728642198#L2392

**Comment by ThomasLecocq** [comment]
should be solved now with a8288e0a4d3dc9c2b247c78b95445a752cb07292
